### DISCUSSION TITLE
Change in VPC.fromlookup?

### DISCUSSION THREAD

**Discussion by daknhh** [aws/aws-cdk#26790] [Q&A]
When try to use VPC.fromLooup with  cdk version 2.92.0 the isolatedSubnets are empty. With cdk version 2.82.0 isolatedSubnets array is set to private subnet, since i cant find any change in the release notes, i am wondering why this is happening?

**Comment by goyertp** [comment]
Same situation here. I also encountered the bug with 2.92.0 and therefore had to refactor some cdk projects. Please update the docs when there are essential changes .

**Comment by peterwoodworth** [comment]
Was this introduced in `2.92.0`? Or could it have been in `2.85.0`? [This](https://github.com/aws/aws-cdk/pull/25958) was the most recent change I could find that I think could have done this

**Reply by daknhh** [reply]
I was searching in all of the lastest releases for the changelog and cant find any information about that
